app/utils/track.js

```diff
@@ -260,5 +262,30 @@ export function trackXClick(adSlot) {
 export function setTrackContext(data) {
   context = data;
 }
+/**
+ * Set GA dimensions
+ * @param {number} index
+ * @param {string} value
+ */
+export function setDimension(index, value) {
+  M.tracker.UniversalAnalytics.setDimension(index, value.toString());
+}
+
+export function trackAffiliateUnit(unit, params) {
+  const campaignId = unit.campaign;
+  const unitId = unit.category;
+  const articleId = params.articleId || 'search-page';
+  const taxonomyId = 'some-concatendated-string-value'; // V TODO
```

Comment:
Hmmm
https://services.wikia-dev.us/knowledge-graph/affiliates/1575417 don't you mean the `tracking` prop from unit that should already be on all units targeted by taxonomy?

Reply:
The values aren't there yet for search.